unlock-app/src/actions/error.js

```diff
@@ -1,6 +1,12 @@
 export const SET_ERROR = 'SET_ERROR'
+export const RESET_ERROR = 'RESET_ERROR'
 
 export const setError = error => ({
   type: SET_ERROR,
   error,
 })
+
+export const resetError = error => ({
```

Comment:
You only seem to be calling this function with no `error`. Why then have it as an argument?

Reply:
And then since it seems to remove all error*s* when I would call it __resetError*s*__.